src/pages/mytask/task-item.tsx

```diff
@@ -60,18 +50,56 @@ export const TaskItem = ({ item }: { item: Task }) => {
                 <Clock className="w-4 h-4" />
                 <span className="text-xs">{formatDistanceToNow(new Date(item.createdAt), { addSuffix: true })}</span>
               </div>
-              <div className="flex flex-row gap-2 pr-4">
-                {item.labels.length > 0 && (
-                  <span
-                    className="inline-flex items-center px-2.5 py-0.5 border rounded-full focus:ring-2 focus:ring-ring focus:ring-offset-2 font-semibold text-xs transition-colors focus:outline-none"
-                    style={{ backgroundColor: getRandomColor() }}
+            </div>
+            <div className="">
+              {item.labels.length > 0 && (
+                <span
+                  className="inline-flex items-center px-2.5 py-0.5 border rounded-full focus:ring-2 focus:ring-ring focus:ring-offset-2 font-semibold text-xs transition-colors focus:outline-none"
+                  style={{ backgroundColor: getRandomColor() }}
+                >
+                  {item.labels[0]}
+                </span>
+              )}
+            </div>
+
+            <div className="flex flex-row justify-between items-center gap-2 pr-4">
+              <div>
+                {item.state === 'closed' ? (
+                  <Button
+                    disabled
+                    className="border-white/80 bg-greyscale-50/8 hover:bg-white/10 border hover:border-opacity-80 w-24 text-white"
+                  >
+                    Closed
+                  </Button>
+                ) : !item.assignees.length ? (
+                  <></>
+                ) : (
+                  // <Button
+                  //   onClick={handleDisclaim}
+                  //   className="border-white/80 bg-greyscale-50/8 hover:bg-white/10 border hover:border-opacity-80 w-24 text-white"
+                  // >
+                  //   Disclaim
+                  // </Button>
+                  <Button
+                    disabled
+                    className="border-white/80 bg-greyscale-50/8 hover:bg-white/10 border hover:border-opacity-80 w-24 text-white"
                   >
-                    {item.labels[0]}
-                  </span>
+                    Claimed
```

Comment:
can user know the difference it is claimed by him/her or by others?

Reply:
In my view, on the My Tasks page, users are likely more concerned with the status of a task—whether it is open or closed—rather than whether it is claimed. This is because the page focuses on their personal tasks.

Reply:
Sorry. I thought I was commented on Task page. No worry.